review.md

```diff
@@ -2,48 +2,72 @@
+
+1. Merge the pull request to accept the contribution.
+2. Flag the pull request for manual review by a [moderator](governance.md#moderator).
+
+## Automatic acceptance
+
+The bot automatically accepts the contribution if the author of the [pull request](https://github.com/r-multiverse/contributions/pulls):
+
+1. Is a [public member](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership) of one of the trusted [GitHub organizations](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations) listed at <https://github.com/r-multiverse/contributions/blob/main/organizations>.
+
+and if the [pull request](https://github.com/r-multiverse/contributions/pulls) itself:
+
+1. Was [created by the GitHub web interface](contributors.md).
+1. Adds new contributed listings to the [`packages` folder](https://github.com/r-multiverse/contributions/tree/main/packages).
+1. Does not add, modify, or delete any other files in <https://github.com/r-multiverse/contributions>.
+
+and if each new [contributed listing](https://github.com/r-multiverse/contributions/tree/main/packages):
+
+1. Is a single line of text with a valid HTTPS URL.
+1. Points to an existing public GitHub/GitLab repository.
+
+and if the contributed GitHub/GitLab repository:
+
+1. Includes a GitHub/GitLab [release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases).
+1. Includes an R package at the top level whose package name is the same as the repository name.
+1. Is listed in the `URL` field of the corresponding CRAN page (if a package with the same name is on CRAN).
+
+and if the R package:
+
+1. Includes a license from the "Recommended licenses" section at the end of this policy.
+1. Does not have an advisory in the [R Consortium Advisory Database](https://github.com/RConsortium/r-advisory-database).
+1. Is not part of the CRAN mirror at <https://github.com/cran>.
+
+## Manual review
+
+R-multiverse [moderators](governance.md#moderator) review [pull requests](https://github.com/r-multiverse/contributions/pulls) that the bot flags for manual review.
+The [moderator](governance.md#moderator) inspects the package for compliance with R-multiverse policies, including but not limited to [Acceptable Use](aup.md), [Terms of Use](terms.md), and [Code of Conduct](conduct.md).
+The moderator accepts the contribution if and only if it complies with all policies.
+
+## Removal
+
+R-multiverse may remove a package from its own repositories at any time if the package violates R-multiverse policies.
+
+## Modification
+
+No R-multiverse staff member (administrator, moderator, or otherwise) may modify a package in R-multiverse without the explicit consent of the owners declared in the license of the package.
```

Comment:
Let's just remove this section altogether. It is so outside the expected behaviour of a repository that I don't think we need to cover this scenario.

Reply:
But it's not unheard of in the traditional repositories by any means. I think folks who are aware of past instances will want to know R-multiverse will never meddle with package source code.

Reply:
A vanishingly small number of people likely know about those instances. I think merely mentioning this is on balance harmful as readers might remember this possibility and might remember seeing it on R-multiverse, even if we say this will never happen.

Reply:
That's fair. And I guess the package licenses and https://github.com/r-multiverse/production/blob/main/LICENSE.note legally prevent us from doing so anyway.

Reply:
Removed in https://github.com/wlandau/r-multiverse.github.io/commit/f6dc363996b03781075829a820a81b48cdf0f4ce

Reply:
Thanks!